packages/functional-tests/tests/oauth/totp.spec.ts

```diff
@@ -88,32 +91,62 @@ test.describe('severity-1 #smoke', () => {
 
       await page.waitForURL(/inline_totp_setup/);
 
-      await expect(
-        signin.page.getByText('Enable two-step authentication')
-      ).toBeVisible();
+      // TODO: Remove conditions after the old flow is cleaned up
+      if (config.featureFlags.updatedInlineTotpSetupFlow) {
+        await expect(
+          signin.page.getByRole('heading', {
```

Comment:
It is generally preferred to add locators in the page model (e.g., functional-tests/pages/inlineTotpSetup.ts) for improved readability. The assertions could then be something like: 
`await expect(inlineTotpSetup.flowHeading).toBeVisible();` 

I do see that existing test file does not follow the preferred pattern, but I would recommend that if you split out new vs old test case, that the new test cases follow the preferred pattern.

The old test cases would not need to be updated as they will be pulled out soon.